### ISSUE TITLE
Option to turn off IDACode plugin in IDA and VSCode

### ISSUE THREAD

**Issue by Qynklee** [ioncodes/idacode#34]
After use IDACode sometimes I got some bug, only way to fix them is close and open IDA and VScode.
I think option to turn off and close socket in IDA and VSCode is useful.

**Comment by ioncodes** [comment]
Can you specify what you mean with "bug" and "fix theme"?

**Comment by Qynklee** [comment]
yes. When I use idacode to debug a python script, script.py (has main() function) import other file in same folder. 
When I disconnect debug process, edit other file (not script.py) and save file, other file will be send to IDA. 
After that, I re-connect debug to IDA and Save script.py to debug, IDACode will be not work.
